### PR TITLE
Add compounds to vapor pressure batch9

### PR DESCRIPTION
**Pull request by sumitnegi7**

### CODE REVIEW

File: src/data/vapor-pressure-constants.ts [hunk 1]
```diff
@@ -2748,6 +2748,891 @@ export const DIPPRVaporPressureDictionary: Record<
...
+    CAS: '10035-10-6',
+    C1: 29.315,
+    C2: -2424.5,
+    C3: -1.1354,
+    C4: 2.38e-18,
+    C5: 6,
+    minimumTemperature: 185.15,
+    pressureAtMinimumTemperature: 29500,
+    maximumTemperature: 363.15,
+    pressureAtMaximumTemperature: 8460000,
```
Comment (sumitnegi7) [inline src/data/vapor-pressure-constants.ts:2764]: Sir @maneetgoyal  during automation from table 2 fields are not consisitent like maximum temp below minimum one is it acceptabale or should i fix it

**Comment by saikat-samanta** [comment]
@sumitnegi7, can you share the automation script.

File: src/data/vapor-pressure-constants.ts [hunk 2]
```diff
@@ -2748,6 +2748,891 @@ export const DIPPRVaporPressureDictionary: Record<
...
+    name: 'Hydrogen bromide',
+    formula: 'BrH',
+    CAS: '10035-10-6',
+    C1: 29.315,
+    C2: -2424.5,
+    C3: -1.1354,
+    C4: 2.38e-18,
+    C5: 6,
+    minimumTemperature: 185.15,
+    pressureAtMinimumTemperature: 29500,
```
Comment (saikat-samanta) [inline src/data/vapor-pressure-constants.ts:2762]: @sumitnegi7 , it changes `2.95E+04` to `29500`. Is it okay?

File: src/data/vapor-pressure-constants.ts [hunk 3]
```diff
@@ -2748,6 +2748,891 @@ export const DIPPRVaporPressureDictionary: Record<
...
+  },
+
+  HydrogenChloride: {
+    name: 'Hydrogen chloride',
+    formula: 'ClH',
+    CAS: '7647-01-0',
+    C1: 104.27,
+    C2: -3731.2,
+    C3: -15.047,
+    C4: 0.0313,
```
Comment (saikat-samanta) [inline src/data/vapor-pressure-constants.ts:2774]: 0.03134

File: src/data/vapor-pressure-constants.ts [hunk 4]
```diff
@@ -2748,6 +2748,891 @@ export const DIPPRVaporPressureDictionary: Record<
...
+    C5: 1,
+    minimumTemperature: 158.97,
+    pressureAtMinimumTemperature: 13500,
+    maximumTemperature: 324.65,
+    pressureAtMaximumTemperature: 8360000,
```
Comment (saikat-samanta) [inline src/data/vapor-pressure-constants.ts:2779]: pressureAtMaximumTemperature: 83560000,

**Comment by sumitnegi7** [comment]
@saikat-samanta leave it then i will review it with table once myself

File: src/data/vapor-pressure-constants.ts (hunk 2 above)
Comment (sumitnegi7) [reply src/data/vapor-pressure-constants.ts:2762]: No that is incorrect

File: src/data/vapor-pressure-constants.ts [hunk 5]
```diff
@@ -2748,6 +2748,891 @@ export const DIPPRVaporPressureDictionary: Record<
...
+  },
+
+  HydrogenFluoride: {
+    name: 'Hydrogen fluoride',
+    formula: 'FH',
+    CAS: '7664-39-3',
+    C1: 59.544,
+    C2: -4143.8,
+    C3: -6.1764,
+    C4: 0.0000142,
```
Comment (saikat-samanta) [inline src/data/vapor-pressure-constants.ts:2804]: @sumitnegi7 , It's rounding the value `000014161`

File: src/data/vapor-pressure-constants.ts [hunk 6]
```diff
@@ -2748,6 +2748,891 @@ export const DIPPRVaporPressureDictionary: Record<
...
+  },
+
+  HydrogenSulfide: {
+    name: 'Hydrogen sulfide',
+    formula: 'H2S',
+    CAS: '7783-06-04',
+    C1: 85.584,
+    C2: -3839.9,
+    C3: -11.199,
+    C4: 0.0188,
```
Comment (saikat-samanta) [inline src/data/vapor-pressure-constants.ts:2819]: 0.018848

File: src/data/vapor-pressure-constants.ts [hunk 7]
```diff
@@ -2748,6 +2748,891 @@ export const DIPPRVaporPressureDictionary: Record<
...
+  },
+
+  MethylAmine: {
+    name: 'Methyl amine',
+    formula: 'CH5N',
+    CAS: '74-89-5',
+    C1: 75.206,
+    C2: -5082.8,
+    C3: -8.0919,
+    C4: 0.00000811,
```
Comment (saikat-samanta) [inline src/data/vapor-pressure-constants.ts:2984]: 0.000008113

File: src/data/vapor-pressure-constants.ts [hunk 8]
```diff
@@ -2748,6 +2748,891 @@ export const DIPPRVaporPressureDictionary: Record<
...
+  },
+
+  '2Methyl2butene': {
+    name: '2-Methyl-2-butene',
+    formula: 'C5H10',
+    CAS: '513-35-9',
+    C1: 83.927,
+    C2: -5640.5,
+    C3: -9.6453,
+    C4: 0.0000111,
```
Comment (saikat-samanta) [inline src/data/vapor-pressure-constants.ts:3089]: 0.000011121
Check other `C4` property

**Pull request closed without merging**